pytorch_lightning/plugins/environments/torchelastic_environment.py

```diff
@@ -24,8 +24,11 @@
 
 class TorchElasticEnvironment(ClusterEnvironment):
 
-    def __init__(self):
-        super().__init__()
+    @staticmethod
+    def is_using_torchelastic() -> bool:
```

Comment:
Why don't we have something similar for slurm?

Reply:
we have, it's in the accelerator connector. and I will do a follow up. didn't want to make the PR larger